package.json

```diff
@@ -19,6 +19,7 @@
     "multer": "^1.4.5-lts.1",
     "mysql2": "^3.7.0",
     "sequelize": "^6.35.2",
+    "ts-node": "^10.9.2",
```

Comment:
why adding ts-node if you are not using it ?
Also it is better to be in devDependencies section. The same for typescript package

Reply:
I added it during debugging  and forgot to remove it